.teamcity/components/settings.kt

```diff
@@ -143,6 +143,9 @@ var serviceTestConfigurationOverrides = mapOf(
         // Private DNS Resolver is only available in certain locations
         "privatednsresolver" to testConfiguration(locationOverride = LocationConfiguration("eastus", "westus3", "westeurope", true)),
 
+        // Qumulo asked to use canary env for testing
+        "qumulo" to testConfiguration(locationOverride = LocationConfiguration("centraluseuap", "eastus2euap", "westeurope", true)),
```

Comment:
we need to use real production environments - the behaviour in canary can differ from other environments, so whilst we can test there it shouldn't be 2 of the 3 locations - can we make `westeurope` the default, one other production region and then one staging env?